contents/teams/product-analytics/objectives.mdx

```diff
@@ -1,32 +1,26 @@
-### Q4 2024 objectives
-
-1.  Legacy 0x - Thomas owning
-    - All insights are `query`-based
-    - `rm -rf posthog/queries/`
-    - Insight query is stored in the URL for instant sharing of unsaved insights
-
-2. First-class environments - Michael owning
-   - Environments are rolled out to everyone
-
-3. Threshold-based alerts - Anirudh owning
-    - All users get to set up alerts on PostHog metrics
-    - Alerts are launched with Marketing, and positioned into paid plans
-    - We're using alerts ourselves for key trends and funnels
-
-4. Flexible funnels - Sandy owning
-    - UDF funnels rolled out to everyone, unlocking arbitrary complexity
-    - Allowing parts of funnels to be unordered. No more separation between regular, unordered, and strict funnels
-    - Optional funnel steps
-
-5. 10x onboarding and ease of discovering features - Anirudh and Sandy owning
-    - We run an exercise in implementing PostHog in a hobby project. What stands out as painful or unclear?
-
-6.  User delight 2x - support rotation owning
-    -  Support is 1 hero + 1 sidekick
-    -  Every week support folks ship those small requested features (see [list of requests so far](https://docs.google.com/spreadsheets/d/1-lXjfzo6313Ai2vMf6smmoA0zghitBQTRD4lPgZ-XqA/edit?gid=0#gid=0))
-    -  Goal: 90% of tickets fulfill the SLA
-
-7. B2B analytics research - Anna owning
-    - Understand the market and user needs better in order to improve our current Group analytics product next quarter
-
-See [our internal Q4 planning doc](https://docs.google.com/document/d/15pATJU0rYxl5LN9LRLfzPVFmcYT1OIFl1cOQwZmnjKA/edit) for the discussion that led us here.
+### Q1 2025 objectives
+
+1. Main Goal: Make it a no-brainer for customers to use our insights
+    - Reduce the reliance on HogQL to do analysis
```

Comment:
I think this is mainly about areas where HogQL is used as a crutch/workaround and better UI should exist - don't want to discourage HogQL insights in general. Maybe:
```suggestion
    - Reduce usage of HogQL as a workaround/crutch for missing UI features
```